frontend/packages/console-shared/src/components/lists/DetailPropertyList.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { List } from '@patternfly/react-core';
+
+import './DetailPropertyList.scss';
+
+const DetailPropertyList: React.FC<React.ComponentProps<typeof List>> = ({
```

Comment:
you can use `ListProps` directly
```suggestion
const DetailPropertyList: React.FC<ListProps> = ({
```

Reply:
Done